roles/postgres/tasks/main.yml

```diff
@@ -41,6 +41,17 @@
     address: "0.0.0.0/0"
     method: "md5"
 
+- name: Configure pg_hba to accept connection from local trento-web and trento-wanda services
```

Comment:
We don't need to run this when install_method is docker but it doesn't hurt either, wdyt?

Reply:
Why don't you put a `when` condition?
You could put the same default value to `installation_method` in the defaults, that will be overriden when the use chooses the other option.

Anyway, here, you make the assumption that the database and the rpm packages are running in the same host, which is not true. They can be running in different hosts.
Maybe you can get the address of the machine where web/wanda are installed from the facts or hosts data (i'm pretty sure we can do something like that with ansible)